spatial-lib/src/cmr/spatial/point.clj

```diff
@@ -168,7 +169,14 @@
            :lat-rad lat-rad
            :options options
            :geodetic-equality geodetic-equality
-           not-found)))
+           not-found))
+  
+  java.lang.Iterable
```

Comment:
When I was trying to make a test for the change, there was an error that kept being thrown about iterator() being an abstract method in the Point deftype. When I looked into it, clojure.lang.IPersistentMap inherits the method iterator() from java.lang.Iterable but that hadn't been defined in the deftype so I had to add that in. I don't know where it might be needed now, but I figured I'd leave it in despite no longer trying to make a test case

Reply:
removed